OpenRA.Mods.Common/UpdateRules/Rules/ScaleDefaultModHealth.cs

```diff
@@ -0,0 +1,118 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using OpenRA.Primitives;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class ScaleDefaultModHealth : UpdateRule
```

Comment:
I'm not sure how useful this is as it is currently written - we do need to have rules keyed on mod ID for the resource center, but this won't run at all for other mods.  It would be better if we could have a base `ScaleModHealth` superclass that takes the scaling factor in the ctor, a pair of `ScaleModHealthBy10` and `ScaleModHealthBy100` subclasses that can be called manually by any mod, and then a `ScaleDefaultModHealth` subclass that then uses `ModScales` to pick the scaling value or skips if non-default.